crates/ruff_server/src/server/api/requests/execute_command.rs

```diff
@@ -40,6 +41,15 @@ impl super::SyncRequestHandler for ExecuteCommand {
         let command =
             Command::from_str(&params.command).with_failure_code(ErrorCode::InvalidParams)?;
 
+        if let Command::Debug = command {
```

Comment:
nit: slight preference to just use `==` instead (`if command == Command::Debug`)

Reply:
The advantage of `let` is that we don't need to add a `PartialEq` implementation to `Command` - but that's fine, I'm not that opinionated on this.